libqtile/widget/image.py

```diff
@@ -58,45 +58,26 @@ def _configure(self, qtile, bar):
             raise ValueError("Filename not set!")
 
         self.filename = os.path.expanduser(self.filename)
```

Comment:
That makes sense. For this widget I really just tried to change the minimum amount from the baseline to get it switched over to use `images.Img()`, so that it can maybe more easily be used as an example for modifying the other widgets.